@navikt/core/react/src/progress-bar/ProgressBar.tsx

```diff
@@ -0,0 +1,129 @@
+import cl from "clsx";
+import React, { HTMLAttributes, forwardRef, useEffect, useState } from "react";
+
+interface ProgressBarPropsBase
+  extends Omit<HTMLAttributes<HTMLDivElement>, "role"> {
+  /**
+   * Changes height of progressbar.
+   * @default "medium"
+   */
+  size?: "large" | "medium" | "small";
+  /**
+   * Value of progressbar.
```

Comment:
```suggestion
   * Value of progress bar.
```
Eventuelt, for å følge "mønsteret" på valueMin og valueMax:
```suggestion
   * Current progress.
```